getsolus/packages
# nuget: Update to 6.0.0.280

**Summary**

Updated to a version with a working nuget build repository.

new features: Source mapping, .net 6 support, warns about package vulnerabilities.

Full release notes available [here](https://devblogs.microsoft.com/nuget/announcing-nuget-6/)

Solves #2635 

**Test Plan**

 - Pack a .net class project
 - install a nuget package
 - confirm godot builds with new nuget version

**Checklist**
